src/components/user/PersonalProfileEditForm.js

```diff
@@ -44,6 +55,17 @@ export default function PersonalProfileEditForm() {
       });
     } else {
       try {
+        // If there's a profile picture, upload it first
+        if (formData.profilePicture) {
+          const formDataImage = new FormData();
+          formDataImage.append('profile', formData.profilePicture);
+
+          // Make the API call to upload the image
+          const imageResponse = await axiosInstance.post(
+            `/users/${user?._id}/uploadProfile`,
```

Comment:
Same as above; you'll need to change this to refer to `userData`

```suggestion
            `/users/${userData?._id}/uploadProfile`,
```

Reply:
Thank you ❤️

Reply:
Akos , 
I have made the changes according to our chat above  but I still have errors, this time about axios request. 
<img width="436" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/88336329/63534b57-dd9d-4634-a47f-c08e82e7ec4e">